syft/frameworks/torch/hook/hook.py

```diff
@@ -492,6 +493,150 @@ def _hook_crypten(self):
         new_func = get_hooked_crypten_func("load", crypten_load)
         setattr(crypten, "load", new_func)
 
+    def _hook_crypten_module(self):
+        """Overloading crypten.nn.Module with PySyft functionality, the primary module
+           responsible for core ML functionality such as Neural network layers and
+           loss functions.
+           It is important to note that all the operations are actually in-place.
+        """
+        import crypten
+
+        def _check_encrypted(model):
+            if model.encrypted:
+                raise RuntimeError("Crypten model must be unencrypted to run PySyft operations")
+
+        crypten.nn.Module._check_encrypted = _check_encrypted
+
+        def module_is_missing_grad(model):
```

Comment:
Q: Should be ```module``` or ```model```?

Reply:
I kept the same naming as the method hooked in torch.nn.Module